src/me/drton/jmavsim/Simulator.java

```diff
@@ -306,7 +306,7 @@ private AbstractMulticopter buildMulticopter() {
         I.m11 = 0.005;  // Y
         I.m22 = 0.009;  // Z
         vehicle.setMomentOfInertia(I);
-        vehicle.setMass(1.0); // mass of the vehicle, too light the vehicle will start climbing on arm
+        vehicle.setMass(0.8);
```

Comment:
Can you explain the reason for this change? All our sim vehicles are close to 1 KG and the consistency helps switching between simulators.